test/configCases/css/minimize-dead-rules/style.css

```
@@ -0,0 +1,57 @@
+   same rule makes it dead even where the two blocks differ otherwise */
+.t09 { width: 1px; .inner { color: red } }
+.between9 { color: teal }
+.t09 { height: 2px; .inner { color: red } }
+
+/*! and the same selector under another rule is another rule */
+.t10a { .inner { color: red } }
+.t10b { .inner { color: red } }
+
+/*! a raw passthrough the parser kept verbatim stands among the rules it
+   separates, and is counted among them */
+@media (min-width: 1px) { *zoom: 1; .t11 { .inner { color: red } } }
```

Stylelint (17.14.0)

[error] Invalid position for declaration (no-invalid-position-declaration)

(no-invalid-position-declaration)